### PR TITLE
documentation for IBFT's newly added RPC api

### LINKED ISSUE #61: Doc new RPC APIs for IBFT and Clique
Dev PR: https://github.com/ConsenSys/quorum/pull/1113

### PR DESCRIPTION
**Pull request by amalrajmani**
documentation for IBFT's newly added RPC api - `istanbul.status` and `istanbul.isValidator`

## Fixed Issue(s)

fixes #61. (geth Clique API is linked to and so does not need to be updated on our doc site.)

### CODE REVIEW

File: docs/Reference/Consensus/IBFT-RPC-API.md [hunk 1]
```diff
@@ -169,3 +169,39 @@ istanbul.getSignersFromBlockByHash(blockHash)
 - `hash`: `String` - The retrieved block's hash
 - `author`: `String` - The address of the block proposer
 - `committers`: `[]String` - The list of all addresses whose seal appears in this block
+
+### istanbul.status
+
+Returns the signing status of blocks for a given block range. If start block number and end block number are not provided then it returns the status of last 64 blocks by default.
```
Comment (alexandratran) [inline docs/Reference/Consensus/IBFT-RPC-API.md:175]: Suggestion: "If the start block and end block numbers are not provided, it returns the status of the last 64 blocks by default."

File: docs/Reference/Consensus/IBFT-RPC-API.md [hunk 2]
```diff
@@ -169,3 +169,39 @@ istanbul.getSignersFromBlockByHash(blockHash)
...
+#### Parameters
+
+- `Number` - start block number
+- `Number` - end block number
+
+#### Returns
+
+`Object`
+
+- `numBlocks`: `Number` - no of blocks for which sealer activity is retrieved
```
Comment (alexandratran) [inline docs/Reference/Consensus/IBFT-RPC-API.md:190]: "number of blocks..."

File: docs/Reference/Consensus/IBFT-RPC-API.md [hunk 3]
```diff
@@ -169,3 +169,39 @@ istanbul.getSignersFromBlockByHash(blockHash)
...
+- `sealerActivity`: `map[string]Number` - key is the validator and value is the number of blocks sealed by the validator
+
+### istanbul.isValidator
+
+Returns true if this node was the validator for the given blockNumber. If blockNumber is not provided it takes the latest block number by default.
```
Comment (alexandratran) [inline docs/Reference/Consensus/IBFT-RPC-API.md:195]: - "`true`" and "`blockNumber`" should be surrounded by backticks (`).
- Change "was" to "is".

File: docs/Reference/Consensus/IBFT-RPC-API.md [hunk 4]
```diff
@@ -169,3 +169,39 @@ istanbul.getSignersFromBlockByHash(blockHash)
...
+istanbul.isValidator(blockNumber)
+```
+
+#### Parameters
+
+- `Number` - block number
+
+#### Returns
+
+- `Boolean` - validator status of this node for the given blockNumber.
```
Comment (alexandratran) [inline docs/Reference/Consensus/IBFT-RPC-API.md:207]: "`blockNumber`" should be surrounded by backticks.

**Review by alexandratran — COMMENTED** [review]
Added comments.

File: docs/Reference/Consensus/IBFT-RPC-API.md (hunk 1 above)
Comment (amalrajmani) [reply docs/Reference/Consensus/IBFT-RPC-API.md]: done

File: docs/Reference/Consensus/IBFT-RPC-API.md (hunk 3 above)
Comment (amalrajmani) [reply docs/Reference/Consensus/IBFT-RPC-API.md]: done

File: docs/Reference/Consensus/IBFT-RPC-API.md (hunk 4 above)
Comment (amalrajmani) [reply docs/Reference/Consensus/IBFT-RPC-API.md]: done

**Review by alexandratran — APPROVED** [review]
Looks good to me.

**Review by bgravenorst — APPROVED** [review]
LGTM

**Pull request merged**